src/calculateRentalCost.js

```diff
@@ -29,6 +29,17 @@
 
 function calculateRentalCost(days) {
   // write code here
-}
+  let number = 0;
+
+  if (days < 3) {
+    number = days * 40;
+  } else if
+  (days < 7) {
+    number = (days * 40) - 20;
+  } else {
+    number = (days * 40) - 50;
+  }
 
+  return number;
+}
```

Comment:
let's not have magic numbers in the code. at the moment i dont understand what 20, 40 and 50 are
You have repeated `days * 40` a few times. maybe put it in a variable?

Reply:
implemented

Reply:
and yes one variable for this `days * dailycost`
`let totalCost = days * dailycost`

this looks good right 
 `actualCost = (totalCost) - disountForThreeDays `
 (may be change `number` variable as well)

Reply:
implemented